sys-devel/llvm/patches/libunwind-12.0.1.patchset

```diff
@@ -0,0 +1,416 @@
+From 3e3979c2ae83dd20ead2ae41dff99090a982d274 Mon Sep 17 00:00:00 2001
+From: X512 <danger_mail@list.ru>
+Date: Wed, 16 Mar 2022 07:04:18 +0900
+Subject: libroot: implement Itanium ABI unwind functions by using LLVM
```

Comment:
Well that's not what this change is doing. How much of these changes are really necessary anyway?

Reply:
This is a patch extracted from X512's work and I'm keeping everything intact.

The patch adds support for libunwind signal frame unwinding on Haiku.

Reply:
I've added a more meaningful commit message.